src/services/user-group/services/course-copy-service.js

```diff
@@ -79,8 +83,7 @@ class CourseCopyService {
 			userId: params.account.userId,
 			shareToken: lesson.shareToken,
 		}))).catch((err) => {
-			logger.warning(err);
-			throw new GeneralError('Can not copy one or many lessons.', err);
+			throw new BadRequest('Can not copy one or many lessons.', err);
```

Comment:
even more why? This clearly is a server error, not a a problem of the request...